src/concepts/config-intro.md

```diff
@@ -4,59 +4,84 @@ The Zilla runtime configuration defines the [`bindings`](../reference/config/ove
 
 - `proxy` - Handles the translate or encode behaviors between components.
-- `server` - Exists to decode a protocol on the inbound network stream, producing higher level application streams for each request.
+- `server` - Exists to decode a protocol on the inbound network stream, producing higher-level application streams for each request.
 - `client` - Receives inbound application streams and encodes each as a network stream.
-- `remote_server` - Exists to adapt `kafka` topic streams to higher level application streams. Read more in the [kafka-grpc binding](../reference/config/bindings/binding-kafka-grpc.md#summary).
+- `remote_server` - Exists to adapt `kafka` topic streams to higher-level application streams. Read more in the [kafka-grpc binding](../reference/config/bindings/binding-kafka-grpc.md#summary).
 - `cache_client` & `cache_server` - Combined provide a persistent cache of `kafka` messages per `topic` `partition` honoring the `kafka` `topic` configuration for message expiration and compaction. Read more in the [kafka binding](../reference/config/bindings/binding-kafka.md#cache-behavior).
 
 ### Routes
 
-A Route's matching conditions are defined in terms specific to each `binding` type with some common design principles. As each incoming data stream arrives, the binding follows its configured `routes` to reach an `exit` binding or rejects the stream if no routes are viable.
+A Route's matching conditions are defined in terms specific to each `binding` type with some common design principles. As each incoming data stream arrives, the binding follows its configured `routes` to reach an `exit` binding or rejects the stream if no routes are viable. Bindings can define a default `exit` for streams that do not match any route.
 
 ::: info Order Matters
-Messages on a data stream will use the first route with a matching `when` clause in the list. Put the more specific routes first and generic routes last. Often, that looks like a wildcard route last to catch any messages that don't have a previous matching route.
+Messages on a data stream will use the first route with a matching `when` condition in the list. Order the more specific routes first and generic routes last. Often, that looks like a wildcard route last to catch any messages that don't have a previous matching route.
 :::
 
-### Route Exit
-
-A route exists to direct messages on the stream to a desired exit point.
-
 ### When a Route matches
 
-Each route can list conditions (any match) to match entry data streams. Attributes like headers, metadata, source, destination, etc., are used `when` determining the correct `exit` for a message.
-
-#### Path and Method matches
-
-Patterns for `path` and `method` routing require an exact match.
+Each route can list one or many conditions to match data streams. Attributes like headers, metadata, source, destination, etc., are used `when` determining the correct `exit` for a message.
```

Comment:
Note we are routing streams, not individual messages, then messages flow on streams. This means the information in the beginning of the stream triggers routing decisions, not per message.